hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -1643,6 +1643,7 @@ message CreateSnapshotRequest {
   required string volumeName = 1;
   required string bucketName = 2;
```

Comment:
Unrelated to this PR, but we should mark these fields as `optional` before we merge snapshot branch to master. Filed https://issues.apache.org/jira/browse/HDDS-7600